fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AstToStringBuilder.java

```diff
@@ -313,7 +313,9 @@ public String visitSetStatement(SetStmt stmt, Void context) {
                 if (setVar instanceof SystemVariable) {
                     SystemVariable systemVariable = (SystemVariable) setVar;
                     String setVarSql = "";
-                    setVarSql += systemVariable.getType().toString() + " ";
+                    if (systemVariable.getType() != null) {
```

Comment:
At what situation, this type will be null?

Reply:
set @@sql_select_limit=DEFAULT

Reply:
set session variables with using @@